src/rest-server/src/models/v2/group.js

```diff
@@ -349,7 +349,7 @@ const createDefaultAdminUser = async () => {
 const deleteNonexistVCs = async () => {
   try {
     const groupInfoList = await getAllGroup();
-    const vcList = await vcModel.prototype.getVcListAsyc();
+    const vcList = await vcModel.list();
```

Comment:
So the list function is an async or promise?

Reply:
Yes, it's promise